### ISSUE TITLE
Logging configuration not apply condition status != '200'

### ISSUE THREAD

**Issue by tomcamin-dyc** [gravitee-io/issues#9034]
## Describe the bug
Hi i try to catch error on our call api to understand api call for all error 500 and other. I don't want to catch description of status 200 api call because work well (no need to store something good).

## To Reproduce
Steps to reproduce the behaviour:
1. Go to api definition
2. Click on Logs
3. Click on Configure the logging
4. Set conditions like {#response.status != 200}

## Expected behaviour
When i got response status == 500 i must find into log detail all information (body sent and response json with error message)

## Current behaviour
Today when i receive response with status 500 i get exactly the same basic data catch like if it's a 200 (condition not detected)

## Useful information
<img width="870" alt="Screenshot 2023-04-24 at 19 49 27" src="https://user-images.githubusercontent.com/120566734/234077902-8a81da7c-3db4-4c79-acf6-3402c583c078.png">

## Environment
version 3.20.4

**Comment by vdusauto** [comment]
I'd be happy to hear from this topic as it happened not to be possible in the past if I remember correctly (Decision to log or was made during on request phase, sounds right ?). If this has changed and is indeed a bug, very good news !

**Comment by gaetanmaisse** [comment]
@vdusauto you're right and it has not changed, evaluation of the condition is done at the beginning of the request processor chain, so only attributes available at that time can be used (it means nothing from the response or set by policies).

We won't be able to change this behavior anytime soon, but at least we could add a small hint in the Condition section to explain that.
